### PR TITLE
[MIG][14.0] crm_iap_lead_enrich: migration script

### PR DESCRIPTION
**Pull request by tranngocson1996**
This PR
-------
Migration done

### CODE REVIEW

**Comment by tranngocson1996** [comment]
@legalsylvain PR has split
cc: @pedrobaeza

File: openupgrade_scripts/scripts/crm_iap_lead_enrich/14.0.1.1/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    env.ref("crm_iap_lead_enrich.mail_message_lead_enrich_with_data").unlink()
```
Comment (MiquelRForgeFlow) [inline openupgrade_scripts/scripts/crm_iap_lead_enrich/14.0.1.1/post-migration.py]:
```suggestion
    openupgrade.delete_records_safely_by_xml_id(
        env, ["crm_iap_lead_enrich.mail_message_lead_enrich_with_data"]
    )
```

Comment (tranngocson1996) [reply openupgrade_scripts/scripts/crm_iap_lead_enrich/14.0.1.1/post-migration.py]: @MiquelRForgeFlow Done, but why i still see many modules using `unlink`

Comment (MiquelRForgeFlow) [reply openupgrade_scripts/scripts/crm_iap_lead_enrich/14.0.1.1/post-migration.py]: which modules?

**Review by MiquelRForgeFlow — APPROVED** [review]

File: openupgrade_scripts/scripts/crm_iap_lead_enrich/14.0.1.1/post-migration.py (hunk 1 above)
Comment (tranngocson1996) [reply openupgrade_scripts/scripts/crm_iap_lead_enrich/14.0.1.1/post-migration.py]: Looks like I misread v13. Sr

**Pull request merged**